src/org/gk/gkCurator/CuratorActionCollection.java

```diff
@@ -2191,11 +2191,13 @@ private void createInstance() {
 		InstanceUtilities.sortSchemaClasses(schemaClasses);
 		SchemaClass schemaClass = curatorFrame.getSchemaView().getSchemaPane().getSelectedClass();
 
-		// get list of uneditable attributes,
-		java.util.List uneditableAttNames = AttributeEditConfig.getConfig().getUneditableAttNames();
+		// get list of uncreatable classes.
+		java.util.List uncreatableClassNames = AttributeEditConfig.getConfig().getUncreatableClassNames();
+		System.out.println(uncreatableClassNames);
```

Comment:
Please delete this line.

Reply:
Whoops! I'll be careful not to leave those in the future —